_posts/en/newsletters/2022-07-06-newsletter.md

```diff
@@ -0,0 +1,194 @@
+---
+title: 'Bitcoin Optech Newsletter #207'
+permalink: /en/newsletters/2022/07/06/
+name: 2022-07-06-newsletter
+slug: 2022-07-06-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+- **Long-term block reward funding:** in a thread on the Bitcoin-Dev
+  mailing list ostensibly about [covenants][topic covenants], it was
+  noted that Bitcoin's long term security currently depends on
+  demand for block space.   That demand must generate transaction
+  fees to pay for Proof of Work (PoW) in excess of what an attacker
+  would be willing to purchase to disrupt Bitcoin users.  Developer
+  Peter Todd [pointed out][todd subsidy] that this dependency could be
+  removed if the Bitcoin protocol were modified to include a perpetual
+  subsidy.  Several respondents indicated that they thought the system
+  is better off without a perpetual subsidy, while others looked for
+  alternatives or apparent equivalencies such as demurrage.
```

Comment:
It would be nice to have a short summary of "demurrage" or helpful ML (or other) link.